### ISSUE TITLE
Mutiple opened tabs don't show properly

### ISSUE THREAD

**Issue by niudai** [microsoft/terminal#2134]
# Environment

```none
Windows build number: 10.0.18362.239[run "ver" at a command prompt]
Windows Terminal version 0.2.1831.0:

Any other software? No
```

# Steps to reproduce

material. -->
Opened one WSL terminal tab with a relatively long path like this:

![image](https://user-images.githubusercontent.com/44165182/62001710-04928f00-b129-11e9-9292-ae974dcc4369.png)

The path is too long to give other opened tabs space to show.

I opened another WSL tab, but I cannot see it in the tab bar, only when I maximize the window size can I see it.

## normal size

![image](https://user-images.githubusercontent.com/44165182/62001733-74a11500-b129-11e9-81a2-11ea31aa7709.png)

## maximum  size
![image](https://user-images.githubusercontent.com/44165182/62001722-48859400-b129-11e9-90d0-64750d945afc.png)

# Expected behavior

The tab with relatively long path should shorten itself to give other tabs enough space to show, like in vscode or vs.

Actually this could not be a bug, but may be an issue or feature to be considered.

**Comment by imjasonmiller** [comment]
This might be related to #608 and #1358, where support for a custom tab title was added with the `tabTitle` setting. I have shortened mine, as can be seen [here](https://user-images.githubusercontent.com/12190184/62066481-b87f4180-b231-11e9-9167-71a873b4f15e.png). It doesn't appear to support a dynamic titles at the moment, but there was some talk of that in #1320. On a side note, it looks like you are running an older build which might not support `tabTitle` yet.
